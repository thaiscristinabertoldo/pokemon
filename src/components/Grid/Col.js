import styled from 'styled-components'

export const Col = styled.div`
  flex: ${(prop) => prop.size};
  margin: ${(prop) => prop.margin};
  width: ${(prop) => prop.width};
`

Col.defaultProps = {
  size: 1,
  margin: '10px 20px',
  width: 'inherit',
}
