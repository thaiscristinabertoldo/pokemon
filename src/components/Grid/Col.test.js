import React from 'react'

import { render, cleanup } from '@testing-library/react'

import { Col } from './Col'

afterEach(cleanup)

describe('Col', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<Col />)

    expect(asFragment(<Col />)).toMatchSnapshot()
  })
})
