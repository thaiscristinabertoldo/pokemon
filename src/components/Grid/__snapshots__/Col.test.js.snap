// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm fmlA-dn"
    size="1"
    width="inherit"
  />
</DocumentFragment>
`;
